recipes/goofit/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set version = "2.2.2" %}
+{% set name = "goofit" %}
+
+{% set goofit_proc_version = "1.0.0" %}
+{% set goofit_proc_type = "cpu" %}  # [not linux]
+{% set goofit_proc_type = "cpu" if cuda_compiler_version == "None" else "gpu" %}  # [linux]
+
+package:
+  name: goofit-split
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 82884fd3021555da0953776f81199cbea60287f6bca87b0ecdaa5dd212692c67
+  
+build:
+  skip: true  # [not linux]
+  number: 0
+
+outputs:
+  - name: goofit-proc
+    version: "{{ goofit_proc_version }}"
+    build:
+      number: 0
+      string: "{{ goofit_proc_type }}"
+    test:
+      commands:
+        - exit 0
+    about:
+      home: https://github.com/conda-forge/goofit-feedstock
+      license: BSD-3-Clause
+      license_family: BSD
+      summary: A meta-package to select CPU or GPU GooFit build.
+  - name: goofit
+    version: {{ version }}
+    build:
+      number: 0
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - {{ compiler("cuda") }}  # [linux and cuda_compiler_version != "None"]
+        - cmake
+      host:
+        - python
+        - scikit-build
+        - pip
+        - llvm-openmp  # [c_compiler == "clang"]
```

Comment:
See https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp

Reply:
Thanks! I was looking for that but couldn't find it.